app/auth/authV2/actions/RetrieveClientData.scala

```diff
@@ -44,8 +44,15 @@ class RetrieveClientData @Inject()(sessionDataService: SessionDataService,
       .fromRequestAndSession(request, request.session)
 
     sessionDataService.getSessionData(useCookie = true).map {
-      case Right(sessionData) => Right(ClientDataRequest(sessionData.mtditid,None, None, sessionData.nino, sessionData.utr,
-        getBooleanFromSession(SessionKeys.isSupportingAgent), getBooleanFromSession(SessionKeys.confirmedClient)))
+      case Right(sessionData) => Right(ClientDataRequest(
+        sessionData.mtditid,
+        r.session.get(SessionKeys.clientFirstName),
+        r.session.get(SessionKeys.clientLastName),
+        sessionData.nino,
+        sessionData.utr,
+        getBooleanFromSession(SessionKeys.isSupportingAgent),
```

Comment:
not sure if we have to store this anywhere in the session as this is expected to be resolved by auth everytime?